acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -290,6 +292,51 @@ public void submitBlsToExecutionChange(
     httpClient.post(getRestApiUrl(), "/eth/v1/beacon/pool/bls_to_execution_changes", body);
   }
 
+  public void waitForBlsToExecutionChangeEventForValidator(int validatorIndex) {
+    if (maybeEventStreamListener.isEmpty()) {
+      startEventListener(List.of(EventType.bls_to_execution_change));
+    }
```

Comment:
This seems like a weird thing to do - if we aren't already subscribed we'll almost certainly have missed the event and the test will fail.

Reply:
That's true... :)